easybuild/easyconfigs/r/RStudio-Server/RStudio-Server-1.3.1093-foss-2020a-Java-11-R-4.0.0.eb

```diff
@@ -16,7 +16,15 @@ toolchain = {'name': 'foss', 'version': '2020a'}
 
 source_urls = ['https://github.com/rstudio/rstudio/archive']
 sources = ['v%(version)s.tar.gz']
-checksums = ['af8dc5e71ee934043856938af2e2065d1f1efef08aba37962ed03f6ccb8201bb']
+patches = [
+    'rstudio-1.2.1335.patch',
+    'rstudio-1.3.1093.patch',
```

Comment:
Without this patch the installation was not easily usable, I've added a sanity check command that would have caught this problem